_docs/individual-payments.md

```diff
@@ -36,6 +36,7 @@ The valid `bank_code` values are:
 
 {% capture data-raw %}
 ```
+9 Payment Service Bank Limited: B97
 Access Bank: 044
```

Comment:
@Sylvance  since the content on ngn-bank.md is similar to this let's have this refactor
```

# Nigeria

{% include corridors/ngn-bank.md recipient_type='individual' %}

```
So that we have only one place to amend NGN::Bank Changes